src/components/info-list.jsx

```diff
@@ -10,31 +11,19 @@ const InfoList = ({
   highlightFirstItem,
   noTitles,
 }) => (
-  <div
-    className={classNames(
-      'info-list',
-      { 'info-list--columns': columns },
-      { 'info-list--compact': isCompact },
-      { 'info-list--no-title': noTitles }
-    )}
-  >
+  <div className={classNames('info-list', { 'info-list--columns': columns })}>
```

Comment:
This should probably be a `ul` and the `InfoListItem` in the map should probably be wrapped in a `li`

Reply:
I decided not to go for this because now `DecoratedListItem` is a standalone element and can be used in its on context e.g. we use `DecoratedListItem` in a grid-cell alone in the Supporting Data section in the home page.

Reply:
That's why I suggested putting both `<ul>` and `<li>` markup in the `InfoList` element. I meant it like so:
```ts
<ul className={classNames('info-list', { 'info-list--columns': columns })}>
    {infoData.map(
      // Only draw if there is content
      (item, index) =>
        item.content && (
          <li key={item.title}>
            <InfoListItem
              title={item.title}
              content={item.content}
              highlight={index === 0 && highlightFirstItem}
              compact={isCompact}
              hideTitle={noTitles}
            />
          />
        )
    )}
  </ul>
```
I guess it's not a deal breaker though

Reply:
Oh, sorry, I misunderstood this one. I will do it now!